__tests__/app.test.js

```diff
@@ -395,3 +394,151 @@ describe("PATCH /api/articles/:article_id --- error handling", () => {
 
 
 
+
+
+
+// ======== 07 ==========
+
+describe("POST /api/articles/:article_id/comments", () => {
+  const validInput = { username: "lurker", body: "a valid body" };
+
+  test("200 status code received when calling api correctly", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(validInput)
+      .expect(200);
+  });
+
+  test("returns a single comment object in the format {comment: [{comment-obj}]}", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(validInput)
+      .expect(200)
+      .then((data) => {
+        expect(data.body).toMatchObject({
+          comment: expect.any(Object),
+        });
+        expect(Array.isArray(data.body.comment)).toBe(true);
+        expect(data.body.comment.length).toBe(1);
+      });
+  });
+
+  test("comment returned is valid format", () => {
+    return request(app)
+      .post("/api/articles/2/comments")
+      .send(validInput)
+      .expect(200)
+      .then((data) => {
+        expect(data.body.comment[0]).toMatchObject({
+          comment_id: expect.any(Number),
+          body: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
We can expect the actual article ID here rather than any number

Reply:
Done. I have kept line 435 as is, but added this below:
        expect(data.body.comment[0].article_id).toBe(2);
For future reference, not sure if I could do that specifically on line 435 somehow?